wear/src/main/java/io/homeassistant/companion/android/home/HomeActivity.kt

```diff
@@ -317,11 +384,190 @@ class HomeActivity : ComponentActivity(), HomeView {
         )
     }
 
+    @Composable
+    private fun LoadOtherSection(swipeDismissableNavController: NavHostController) {
+        Column {
+            SetTitle(R.string.other)
+            Chip(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 10.dp),
+                icon = {
+                    Image(asset = CommunityMaterial.Icon.cmd_cog)
+                },
+                label = {
+                    Text(
+                        text = stringResource(id = R.string.settings)
+                    )
+                },
+                onClick = { swipeDismissableNavController.navigate("Settings") },
```

Comment:
Shouldn't you use SCREEN_SETTINGS here?

Reply:
lol yup totally missed it haha, fixed now